codegen/client.go

```diff
@@ -40,19 +40,44 @@ type clientConfig interface {
 	NewClientSpec(
 		instance *ModuleInstance,
 		h *PackageHelper) (*ClientSpec, error)
-	customImportPath() string
-	fixture() *Fixture
 }
 
 // ClientSubConfig is the "config" field in the client-config.yaml for http
 // client and tchannel client.
 type ClientSubConfig struct {
```

Comment:
In retrospect, `ClientThriftConfig` might be more descriptive of what it abstracts over.